dev/tools/update_icons.dart

```diff
@@ -387,15 +385,15 @@ class _Icon {
 ''';
 
   @override
-  String toString() => id;
+  String toString() => id!;
 
   /// Analogous to [String.compareTo]
   int _compareTo(_Icon b) {
     // Sort a regular icon before its variants.
     if (shortId == b.shortId) {
-      return id.length - b.id.length;
+      return id!.length - b.id!.length;
     } else {
```

Comment:
Redudant else statement because there is a `return` inside `if` which ends the execution of a function.

```dart
/// Analogous to [String.compareTo]
int _compareTo(_Icon b) {
  // Sort a regular icon before its variants.
  if (shortId == b.shortId) {
    return id!.length - b.id!.length;
  }
  return flutterId!.compareTo(b.flutterId!);
}
```